privacy-security-questionnaire.md

```diff
@@ -0,0 +1,76 @@
+## Self-Review Questionnaire: Security and Privacy
+
+| 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+This feature is a Document-Policy header that an origin may choose to include in an HTTP resonse when it has no linked resources in the response, to indicate to the user agent that it may optimize resources required to conduct a speculative parse of the response. This will benefit the user in terms of reducing compute requirements on the client, and improve performance of the experience.
```

Comment:
I don't think this answers the question. This seems like a general description of the feature, instead.

In terms of what information this feature exposes to Web sites or other parties, I think the answer is "none"? Like, look at the sub-questions in https://www.w3.org/TR/security-privacy-questionnaire/#purpose .